### PR TITLE
Lestimated var

### PR DESCRIPTION
**Pull request by APolyakova**
Updated to the latest (4.4 dev) and included the LEstimated VaR section.

### CODE REVIEW

File: finance/value-at-risk/main.ipynb [hunk 1]
```diff
@@ -35,7 +35,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Welcome to atoti 0.4.0!\n",
+      "Welcome to atoti 0.4.4.20200928172539!\n",
```
Comment (jbe456) [inline finance/value-at-risk/main.ipynb:38]: doesn't work with 0.4.3?

**Review by jbe456 — COMMENTED** [review]
- one question about using atoti latest rather than 0.4.3
- you should remove the folder `atoti-sbm` from this branch

File: finance/value-at-risk/main.ipynb (hunk 1 above)
Comment (APolyakova) [reply finance/value-at-risk/main.ipynb:38]: yes it's using a new feature  - https://github.com/atoti/atoti/issues/153

**Review by jbe456 — APPROVED** [review]
I cleaned up the extra `.DS_Store` files that were unnecessary.

**Pull request merged**